packages/vats/src/core/boot-psm.js

```diff
@@ -0,0 +1,223 @@
+  startPSM,
+  PSM_MANIFEST,
+} from '@agoric/inter-protocol/src/proposals/startPSM.js';
+// TODO: factor startEconomicCommittee out of econ-behaviors.js
+import { startEconomicCommittee } from '@agoric/inter-protocol/src/proposals/econ-behaviors.js';
+import { ECON_COMMITTEE_MANIFEST } from '@agoric/inter-protocol/src/proposals/core-proposal.js';
+import { makeAgoricNamesAccess, makePromiseSpace } from './utils.js';
+import { Stable, Stake } from '../tokens.js';
+import {
+  addBankAssets,
+  buildZoe,
+  installBootContracts,
+  makeAddressNameHubs,
+  makeBoard,
+  makeVatsFromBundles,
+  mintInitialSupply,
+} from './basic-behaviors.js';
+import * as utils from './utils.js';
+import {
+  bridgeCoreEval,
+  makeBridgeManager,
+  makeChainStorage,
+  publishAgoricNames,
+  startTimerService,
+} from './chain-behaviors.js';
+import { CHAIN_BOOTSTRAP_MANIFEST } from './manifest.js';
+import {
+  startWalletFactory,
+  WALLET_FACTORY_MANIFEST,
+} from './startWalletFactory.js';
+
+/** @typedef {import('@agoric/inter-protocol/src/proposals/econ-behaviors.js').EconomyBootstrapSpace} EconomyBootstrapSpace */
+
+/**
+ * PSM and gov contracts are available as
+ * named swingset bundles only in
+ * decentral-psm-config.json
+ */
+const PSM_GOV_INSTALL_MANIFEST = {
+  [installGovAndPSMContracts.name]: {
+    vatPowers: { D: true },
+    devices: { vatAdmin: true },
+    consume: { zoe: 'zoe' },
+    installation: {
+      produce: {
+        contractGovernor: 'zoe',
+        committee: 'zoe',
+        binaryVoteCounter: 'zoe',
+        psm: 'zoe',
+      },
+    },
+  },
+};
+
+/**
+ * We reserve these keys in name hubs.
+ */
+export const agoricNamesReserved = harden(
+  /** @type {const} */ ({
+    issuer: {
+      [Stake.symbol]: Stake.proposedName,
+      [Stable.symbol]: Stable.proposedName,
+      AUSD: 'Agoric bridged USDC',
+    },
+    brand: {
+      [Stake.symbol]: Stake.proposedName,
+      [Stable.symbol]: Stable.proposedName,
+      AUSD: 'Agoric bridged USDC',
+    },
+    installation: {
+      centralSupply: 'central supply',
+      mintHolder: 'mint holder',
+      walletFactory: 'multitenant smart wallet',
+      contractGovernor: 'contract governor',
+      committee: 'committee electorate',
+      binaryVoteCounter: 'binary vote counter',
+      psm: 'Parity Stability Module',
+    },
+    instance: {
+      economicCommittee: 'Economic Committee',
+      psm: 'Parity Stability Module',
+      psmGovernor: 'PSM Governor',
+    },
+  }),
+);
+
+/**
+ * Build root object of the PSM-only bootstrap vat.
+ *
+ * @param {{
+ *   D: DProxy
+ * }} vatPowers
+ * @param {{
+ *     economicCommitteeAddresses: string[],
+ *     anchorAssets: { denom: string }[],
+ * }} vatParameters
+ */
+export const buildRootObject = (vatPowers, vatParameters) => {
+  // @ts-expect-error no TS defs for rickety test scaffolding
+  const log = vatPowers.logger || console.info;
+
+  const {
+    anchorAssets: [{ denom: anchorDenom }], // TODO: handle >1?
```

Comment:
what does "handle >1" mean?

Reply:
Handle more than one anchor asset; that is: start more than 1 PSM contract, each with its own anchor "denom".

Reply:
It was just about as easy to code it up as answer this, so I did.